tests/test_collection.py

```diff
@@ -0,0 +1,54 @@
+from .test_base import TestBase
+from django.urls import reverse
+
+
+class CollectionTest(TestBase):
+    def test_logged_in_user_can_post_collection(self):
```

Comment:
A `test_logged_out_user_cannot_post_collection` would be helpful to have as well